listenbrainz/webserver/static/js/src/RecentListens.test.tsx

```diff
@@ -978,13 +978,14 @@ describe("Pagination", () => {
       wrapper.setState({
         lastFetchedDirection: "older",
         nextListenTs: 1234567891,
+        // We're not expecting to see this ts as it will be updated by checkListensRange
         previousListenTs: 1234567881,
         listens: [],
       });
-
+      const sortedListens = sortBy(listens, "listened_at");
       const getListensForUserSpy = jest
         .fn()
-        .mockImplementation(() => Promise.resolve(listens));
+        .mockImplementation(() => Promise.resolve(sortedListens.slice(0, 25)));
```

Comment:
I've had to return only 25 items (as expected from the API) because on Node 10, somehow the order of sortedListens changes during execution of the test.
See https://tickets.metabrainz.org/browse/LB-756